README.md

```diff
@@ -1 +1,43 @@
-# Hello_Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+### Project Name
+
+Hello_Microverse
+
+
+## Built With
+
+-CSS
+- Github workflow
+- Linters
+- HTML
+
+
```

Comment:
- [ ] Great job 💪on your effort to use and customize the [Microverse README template](https://github.com/microverseinc/readme-template) as requested by the previous reviewer🙄. However, I noticed that you have removed several sections that were not supposed🙆‍♀️ to be removed. I 🙏kindly request that you please include all the sections in the template, including the table of contents. Please note that the only section that is optional to remove is the `FAQ` section, and you may delete the `live demo` section if you decide not to deploy the application. It would be greatly appreciated👏 if you could add and customize all remaining sections and document them thoroughly. By doing so, you will not only improve your project documentation but also make it easier for other people 😎to understand, use, and even contribute to your project. 🙏Please follow the tips below to improve your README documentation.

- All main headers from the [README template](https://github.com/microverseinc/readme-template) are present in a your repo.
- The only optional section is FAQ
- You  delete the live demo section if the you decide not to deploy the application
- Make sure the you have added the section Future Features, including 1-3 potential features to be added
- You remove the section `description`, i.e. "> `Describe the tech stack and include only the relevant sections that apply to your project`." should be removed and filled in with their custom content
you remove template logo. Optionally, the you can replace it with your own logo you they have one